### ISSUE TITLE
Can I extract the losses of each method from the

### ISSUE THREAD

**Issue by marianogabitto** [ml-stat-Sustech/TorchCP#66]
Hi ! 
   I think the library is great but I want to ask a couple of question for a particular application. I will focus on just one method.

   I will focus on the method from "Training Uncertainty-Aware Classifiers with Conformalized Deep Learning" that proposes to modify the classification loss Lc (for example cross entropy) by a loss Lcp = (1-\lambda) Lc + \lambda Lu . I have a model, a dataset with inputs x, labels y and I am currently training it with a loss Lold = Lc + Lfit, where Lfit is something unrelated to the classifier and a particular trainer. 

   Is there any way of extracting L from TorchCP ? I want to keep my entire code with Lold, model and trainer and just replace Lc by Lcp during a training instance.

Best !
